src/core/Table/hooks/useResizeColumns.tsx

```diff
@@ -0,0 +1,440 @@
+      // lets not respond to multiple touches (e.g. 2 or 3 fingers)
+      if (e.touches && e.touches.length > 1) {
+        return;
+      }
+      isTouchEvent = true;
+    }
+
+    // Setting `width` here because it might take several rerenders until actual column width is set.
+    flatHeaders.forEach((h) => {
+      if (!h.width) {
+        h.width = h.resizeWidth;
+      }
+    });
+
+    const headerIdWidths = getLeafHeaders(header).map((d) => [d.id, d.width]);
+    const nextHeaderIdWidths = nextHeader
+      ? getLeafHeaders(nextHeader).map((d) => [d.id, d.width])
+      : [];
+
+    const clientX = isTouchEvent ? Math.round(e.touches[0].clientX) : e.clientX;
+
+    const dispatchMove = (clientXPos: number) =>
+      dispatch({ type: actions.columnResizing, clientX: clientXPos });
+    const dispatchEnd = () =>
+      dispatch({
+        type: actions.columnDoneResizing,
+      });
+
+    const handlersAndEvents = {
+      mouse: {
+        moveEvent: 'mousemove',
+        moveHandler: (e: MouseEvent) => dispatchMove(e.clientX),
+        upEvent: 'mouseup',
+        upHandler: () => {
+          ownerDocument.removeEventListener(
+            'mousemove',
+            handlersAndEvents.mouse.moveHandler,
+          );
+          ownerDocument.removeEventListener(
+            'mouseup',
+            handlersAndEvents.mouse.upHandler,
+          );
+          dispatchEnd();
+        },
+      },
+      touch: {
+        moveEvent: 'touchmove',
+        moveHandler: (e: TouchEvent) => {
+          if (e.cancelable) {
+            e.preventDefault();
+            e.stopPropagation();
+          }
+          dispatchMove(e.touches[0].clientX);
+          return false;
+        },
+        upEvent: 'touchend',
+        upHandler: () => {
+          ownerDocument.removeEventListener(
+            handlersAndEvents.touch.moveEvent,
+            handlersAndEvents.touch.moveHandler,
+          );
+          ownerDocument.removeEventListener(
+            handlersAndEvents.touch.upEvent,
+            handlersAndEvents.touch.moveHandler,
+          );
+          dispatchEnd();
+        },
+      },
+    };
+
+    const events = isTouchEvent
+      ? handlersAndEvents.touch
+      : handlersAndEvents.mouse;
+    const passiveIfSupported = passiveEventSupported()
+      ? { passive: false }
+      : false;
+    ownerDocument.addEventListener(
+      events.moveEvent,
+      events.moveHandler,
+      passiveIfSupported,
+    );
+    ownerDocument.addEventListener(
+      events.upEvent,
+      events.upHandler,
+      passiveIfSupported,
+    );
+
+    dispatch({
+      type: actions.columnStartResizing,
+      columnId: header.id,
+      columnWidth: header.width,
+      nextColumnWidth: nextHeader?.width,
+      headerIdWidths,
+      nextHeaderIdWidths,
+      clientX,
+    });
+  };
+
+  return [
+    props,
+    {
+      onMouseDown: (e: React.TouchEvent & React.MouseEvent) => {
+        e.persist();
+        e.stopPropagation();
```

Comment:
I see you added this. Is it to prevent text selection?

I also saw you added stopPropagation to the onClick of iui-resizer. Are both needed?

Reply:
It is to prevent sort from firing when resizing columns. onClick is not needed, removed.